Private/Find-AuditingIssue.ps1

```diff
@@ -60,10 +61,13 @@ Invoke-Command -ComputerName `'$($_.dNSHostName)`' -ScriptBlock {
 "@
         }
         if ($_.AuditFilter -match 'CA Unavailable') {
-            $Issue.Issue  = $_.AuditFilter
-            $Issue.Fix    = 'N/A'
+            $Issue.Issue = $_.AuditFilter
+            $Issue.Fix = 'N/A'
             $Issue.Revert = 'N/A'
         }
+        if ($SkipRisk -eq $false) {
```

Comment:
Minor feedback: switches are designed to be checked with `$SkipRisk.IsPresent`. A Boolean true/false also works fine but relies on the _implicit_ behavior of the switch being $true if present.